utils/websocket2serial.py

```
@@ -0,0 +1,234 @@
+    # 1. Start the Zeroconf server and register the service
+    zeroconf_instance = AsyncZeroconf() 
+    local_ip = get_local_ip()
+    logging.info(f"Local IP address detected for Zeroconf: {local_ip}")
+
+    # Optional properties for the service (can be useful for clients)
+    properties = {
+        "api_ver": "0.1",
+        "port": str(WEBSOCKET_PORT),
+        "path" : "/ws",
+        "board_id": f"Ayab Serial to WebSocket Bridge for {serial_port_name} at {baud_rate} baud",
+        "port": str(WEBSOCKET_PORT),
```

Ruff (0.11.9)

Dictionary key literal `"port"` repeated

Remove repeated key literal `"port"`

(F601)